src/scripts/content/planfix.js

```diff
@@ -0,0 +1,38 @@
+/*jslint indent: 2 */
+/*global $: false, document: false, togglbutton: false*/
+'use strict';
+
+togglbutton.render('.taskCard-header-data-blocks .b-green-block-taskAfterLabel', {observe: true}, function (elem) {
```

Comment:
The selector for `existingTag` is failing, so the button keeps getting added repeatedly. Could you change this to `.taskCard-header-data-blocks .b-green-block-taskAfterLabel:not(.toggl)` to ensure the button doesn't get added more than once.